pipit/readers/otf2_reader.py

```diff
@@ -231,14 +237,16 @@ def events_reader(self, rank_size):
             trace.close()  # close event files
 
         # returns dictionary with all events and their fields
-        return {
-            "Timestamp (ns)": timestamps,
-            "Event Type": event_types,
-            "Name": names,
-            "Thread": thread_ids,
-            "Process": process_ids,
-            "Attributes": event_attributes,
-        }
+        return pd.DataFrame(
+            {
+                "Timestamp (ns)": timestamps,
+                "Event Type": event_types,
+                "Name": names,
+                "Thread": thread_ids,
+                "Process": process_ids,
+                "Attributes": event_attributes,
+            }
+        )
```

Comment:
why this change?

Reply:
following what alex does, this removes need for the previous combining of dictionaries, which was done confusingly and was ambiguous